doc/docs/security/selinux.md

```diff
@@ -0,0 +1,25 @@
+<!-- markdownlint-disable-file MD013-->
+# BlueChi's SELinux policy
+
+BlueChi provides a custom SELinux policy, limiting access of the `bluechi-controller` and `bluechi-agent`. It can be installed via
+
+```bash
+dnf install bluechi-selinux
+```
+
+By default, the package adds an allow rule for the `bluechi-agent` to connect to BlueChi's default port 842. If the controller port where the agent is connecting to changes, then an AVC entry is created:
```

Comment:
I'd add some commas for better clarity:

"If the controller port, where the agent is connecting to, changes then an AVC entry is created:"

Reply:
I figured out a way to link to the man page section for the `ControllerPort`. I rephrased and simplified the sentence it a bit.

Reply:
OK